apps/logs.go

```diff
@@ -137,6 +153,9 @@ func streamLogs(logsRawURL string, filter string) error {
 					time.Sleep(time.Second * 1)
 				}
 				continue
+
+			} else if strings.Contains(err.Error(), "use of closed network connect") {
```

Comment:
Isn't it possible to find the root error and test against an error instead of a substring? I am sure you already tried but just want to be sure.